src/Symfony/Bundle/FrameworkBundle/Resources/config/annotations.php

```diff
@@ -23,13 +23,10 @@
 return static function (ContainerConfigurator $container) {
     $container->services()
         ->set('annotations.reader', AnnotationReader::class)
-            ->call('addGlobalIgnoredName', [
-                'required',
-                service('annotations.dummy_registry')->ignoreOnInvalid(), // dummy arg to register class_exists as annotation loader only when required
-            ])
+            ->call('addGlobalIgnoredName', ['required'])
 
         ->set('annotations.dummy_registry', AnnotationRegistry::class)
-            ->call('registerUniqueLoader', ['class_exists'])
+            ->deprecate('symfony/framework-bundle', '6.3', 'The "%service_id%" service is deprecated and will be removed without replacement.')
```

Comment:
I'd suggest just dropping this, nobody can use it today

Reply:
Dropped.